### ISSUE TITLE
Slight visual changes overall

### ISSUE THREAD

**Issue by ppblitto** [dantaeusb-mc/Zetter#2]
### I simply changed some item and GUI textures:

_Canvas item icon:_
![canvas](https://user-images.githubusercontent.com/57157437/123169714-04ec4f80-d450-11eb-8f78-bcf71a149e7a.png)

_Palette item icon:_
![palette](https://user-images.githubusercontent.com/57157437/123169740-0c135d80-d450-11eb-9994-90c8fa87db0f.png)

_Painting GUI Tool icons:_
![painting](https://user-images.githubusercontent.com/57157437/123169778-19c8e300-d450-11eb-880f-ee54af07e2f4.png)

**Comment by ppblitto** [comment]
This should have a "enhancement" label, i just didn't find how to put it. Sorry.

**Comment by dantaeusb** [comment]
Wow! That's nice looking icons and buttons, for sure! 

Thank you! One thought -- while the new canvas icon definitely looks better in texture, I'm not particularly sure about making it isometric. I want this mod to be as close to the vanilla feel as possible, and, as I see, for the painting-related and tool-related icons they try to stick with upfront view. I may be overthinking, but maybe you have some ideas how to make texture as good as you have but keeping it upfront. If not, I'll stick with this variant anyway.

And a big thanks specifically for palette icon -- you really saved the day here because I spent hours drawing the current version and wasn't satisfied with the look. Yours look perfect.

Definitely will include those in the new version!

P.S. forgot to thank you for translations in previous thread, so thanks for that too :)

**Comment by ppblitto** [comment]
Thanks, man. To be frank, I was a little unsure about the canvas too, but I put here anyway. Feel free to decide. This project of yours is really inspiring, so I just want to give some help in anything I can. I can't express how I wanted this to be a implemented feature in the original game... You just created something unique, congratulations. For real.

**Comment by ppblitto** [comment]
Hey, @dantaeusb. Still in the topic, I've made a re-skin of the paints icon:
![paints](https://user-images.githubusercontent.com/57157437/123528067-3ddc2c80-d6bb-11eb-9f53-45b113b0faa2.png)

Regular 16x size:
![paints_regular_size](https://user-images.githubusercontent.com/57157437/123528068-3e74c300-d6bb-11eb-9367-767e0ee3b916.png)

Well, I just though "how could I unify the appearence of this item since there are two distinct recipes to make it, one for the RGB and other for CMYK?" So my idea is that the result is a "magical ink tube" or a "universal paint" that can be made from the two types of triad of dyes.

Once again, feel free to decite if you want to implement that change. If not, you just got a new idea to inspire the remake of the paints icon to a more "minecraft style" type of art.

**Comment by dantaeusb** [comment]
Hi! 

Much appreciated! This one is great, paints icon is another one that was asking for a good reskin. I've just sent a new version (0.9.1) to CurseForge with this icon (+palette and GUI improvement already here). 
However, after testing the canvas icon for a while, I decided to restore the previous canvas icon. While it looks great in inventory, its isometric view breaks the grid when you're combining canvases on the Artist Table, so upfront view does a better job here.

I've also just added you as an artist in CurseForge for credit. This should also allow you to upload images there, so if you have something to showcase — feel free to upload it.

Sharing some complex textures (multi-layer frame texture and easel texture in psd) that are used in this mod in case you have some ideas how to improve those: https://1drv.ms/u/s!Ap5t53K7OPv8hKJHyv5R-akvIsxuxw?e=jtCihA
